ApplicationConfig/Providers/SessionInfo.php

```diff
@@ -29,7 +29,7 @@ class SessionInfo implements Provider
 
     public function __construct(
         SessionInterface $session,
-        CsrfTokenManagerInterface $csrfTokenManager,
+        CsrfTokenManagerInterface $csrfTokenManager = null,
```

Comment:
Technically yes, but it should be OK IMHO. And btw the service should be marked `private`